src/Uno.UI/FeatureConfiguration.cs

```diff
@@ -539,6 +539,11 @@ public static class ScrollViewer
 			/// </summary>
 			public static TimeSpan? AndroidScrollbarFadeDelay { get; set; }
 #endif
+
+			/// <summary>
+			/// Defines the delay of after which the ScrollViewer starts to move to snap points. The default value is 250ms.
+			/// </summary>
+			public static TimeSpan UnoSnapDelay { get; set; } = TimeSpan.FromMiliseconds(250);
```

Comment:
I don't think we need to prefix with Uno in that case. Why do you think it should?

```suggestion
			public static TimeSpan SnapDelay { get; set; } = TimeSpan.FromMiliseconds(250);
```